contrib/systemd-with-workers/system/matrix.target

```diff
@@ -0,0 +1,7 @@
+[Unit]
+Description=Contains matrix services like synapse, bridges and bots
+After=network.target
+AllowIsolate=no
```

Comment:
isn't this the default? Is there any reason to add it here?

Reply:
Assuming you are refering to the `AllowIsolate=no`. Yes, it's the default value according to the [systemd docs](https://www.freedesktop.org/software/systemd/man/systemd.unit.html#AllowIsolate=). However I personally am a friend of doing important things explicitly. The matrix.target is one you would never want to isolate as all non matrix services would be stopped, effectively killing the system.

Thus I deem this important. Take the (admittedly unlikely) case that systemd decides to change the default value. We would still want this to be false/no as this target is designed as a grouping of services rather than an isolatable target.

To be fair even with this set to true an admin would have to do `systemctl isolate matrix.target` in order to cause mayhem, but I'd like to prevent this under any circumstances.

TL; DR: I deem it important that isolated is false and I personally prefer to do important things explicitly rather than implicitly by default values. But yes, it is the default value and I'll leave the final decision up to you. I won't be offended if you tell me to remove it. :slightly_smiling_face:

Reply:
If you feel like this warrants a deeper/longer discussion feel free to pm me in the matrix [@luca:targodan.de](https://matrix.to/#/@luca:targodan.de)

Reply:
> However I personally am a friend of doing important things explicitly

I think you could argue that there are a bunch of other default settings which are just as important, and pretty soon you end up with a unit file a mile long. 

However, I don't have strong feelings here.